### ISSUE TITLE
Sklearn KDE not working as attempted

### ISSUE THREAD

**Issue by sibirrer** [sibirrer/hierArc#7]
https://github.com/sibirrer/hierArc/blob/0a2148460002548992a54f3d9d0276a74e0d9607/hierarc/Likelihood/LensLikelihood/ddt_hist_likelihood.py#L39

This routine originates from https://github.com/shsuyu/H0LiCOW-public/blob/master/H0_inference_code/lensutils.py but did not work with the current sklearn version. @aymgal @martin-millon  do you know whether the notebooks for the sampling of the Wong et al. paper are a jour with the latest versions? In the meantime I replaced it with the scipy gaussian kde which may not have the accuracy we require.

**Comment by aymgal** [comment]
With what I have in hand I cannot answer but @martin-millon sure will.

**Comment by martin-millon** [comment]
It was working for me with lenstronomy 1.3.0. I did not update it to the latest version yet.

**Comment by sibirrer** [comment]
@martin-millon I am not pointing on a lenstronomy dependent routine. It is the sklearn KDE with weights attached, effectively line 185 or 218 in the file on the H0LiCOW repo of the definition likelihood_hist()

**Comment by martin-millon** [comment]
OK, sorry, I am running the script with sklearn v0.21.3. The latest version of sklearn is now 0.22.2 and I did not update the script to the latest version.

**Comment by sibirrer** [comment]
Thank you! I could fix it. The requirement was `scikit-learn` and I installed `sklearn`

**Comment by martin-millon** [comment]
OK, I'll change the requirements in the documentation then.
